doc/releases/changelog-dev.md

```diff
@@ -39,73 +18,16 @@
   [Haldane](https://journals.aps.org/prl/pdf/10.1103/PhysRevLett.61.2015) models on a lattice.
```

Comment:
Suggestion for lines 16-17 (can't suggest on them directly as they're too far from changed lines in this commit):
Remove the space/newline between `[Emery]` and the link, so the embedding works as intended.